src/dapp/libexec/dapp/dapp-mk-standard-json

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+import json
+import subprocess
+import os
+import re
+import fnmatch
+from pathlib import Path
+
+template = open(Path(__file__).parent / "../../resources/jsontemplate.json").read()
```

Comment:
Get this from envvar as well.

Reply:
Really? You can already specify the std json with `DAPP_STANDARD_JSON`, and quite a lot of this file is pretty specific to the example one. I imagine the workflow would be to first generate the boilerplate with `dapp mk-standard-json`, tweak it to your needs and then use this as `DAPP_STANDARD_JSON`. Customizing the template seems overkill